site-new/sidebars.ts

```diff
@@ -12,8 +15,67 @@ import type {SidebarsConfig} from '@docusaurus/plugin-content-docs';
  */
 const sidebars: SidebarsConfig = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{type: 'autogenerated', dirName: '.'}],
-
+  docsSidebar: [
+    {
+      type: "doc",
+      id: "index",
+      customProps: { sidebarHeader: "Docs" },
+    },
+    {
+      type: "category",
+      label: "SDKs",
+      collapsible: false,
+      collapsed: false,
+      items: [
+        {
+          type: "link",
+          label: "Web5 SDK",
+          href: "web5-sdk",
+          className: "web5-icon",
+        },
+        {
+          type: "link",
+          label: "tbDex SDK",
```

Comment:
```suggestion
          label: "tbDEX SDK",
```

The correct capitalization is lowercase `tb` and uppercase `DEX`: `tbDEX`